bin/rl/rl.cpp

```diff
@@ -2369,6 +2357,22 @@ WriteEnvLst
             LstFilesOut->Add("\"");
          }
 
+         // how we translate testinfo into env.lst specific stuffs
+         static const char * const TestInfoEnvLstFmt[] =
```

Comment:
This doesn't need to be static really.  Static local arrays are usually not necessary nor a good idea since they incur a condition and memcpy